src/arch/arm/64/kernel/vspace.c

```diff
@@ -1918,7 +1918,7 @@ exception_t benchmark_arch_map_logBuffer(word_t frame_cptr)
 
     ksUserLogBuffer = pptr_to_paddr((void *) frame_pptr);
 
-    *armKSGlobalLogPDE = pde_pde_large_new(
+    *armKSGlobalLogPDE = pte_pte_page_new(
```

Comment:
Would suggest renaming this one to PTE as well (one more occurrence below)

```suggestion
    *armKSGlobalLogPTE = pte_pte_page_new(
```